contents/blog/feature-flag-best-practices.md

```diff
@@ -0,0 +1,198 @@
+---
+date: 2023-01-03
+title: 8 best practices for feature flags and toggles
```

Comment:
I'm trying to think of ways to make this title standout... the top result in Google is also "8 feature flag best practices" so I think it'll look too similar to get any clicks. Open to suggestions on this one!

Reply:
Some ideas:
- Best practices for feature flag fluency
- Best practices for feature flag mastery
- The definitive guide to feature flag best practices
- How to follow feature flag best practices

Reply:
Getting warmer...

- Master Feature Flags: Best practice, tips and examples
- Feature Flags 101: Best practices, tips and examples

I think a kicker works well here. Thoughts or twists on either of these?

Reply:
I do like fluency because then you get the alliteration. So maybe "Feature flag fluency: Best practice, tips, and examples" ? Master is fine if not. 

I don't think this is a _101_ type post.

Reply:
I think master then. It's ok to use title case for the kicker, btw. Think it works better visually.